src/com/google/javascript/jscomp/AbstractCommandLineRunner.java

```diff
@@ -30,6 +30,9 @@
 import com.google.common.collect.Iterables;
```

Comment:
Can you add a unit test here for the new flag?

https://github.com/google/closure-compiler/blob/master/test/com/google/javascript/jscomp/CommandLineRunnerTest.java